chipsec/config.py

```diff
@@ -366,10 +366,10 @@ def get_common_xml(self):
 
         # Locate all common configuration files
         cfg_files = []
-        cfg_vids = [f.name for f in os.scandir(cfg_path) if f.is_dir() and is_hex(f.name)]
+        cfg_vids = [f for f in os.listdir(cfg_path) if os.path.isdir(os.path.join(cfg_path, f)) and is_hex(f)]
         if vid in cfg_vids:
             root_path = os.path.join(cfg_path, vid)
-            cfg_files.extend([config_data(vid, None, f.path)
-                             for f in sorted(os.scandir(root_path), key=lambda x: x.name)
-                             if fnmatch(f.name, '*.xml') and fnmatch(f.name, 'common*')])
+            cfg_files.extend([config_data(vid, None, os.path.join(root_path, f))
+                             for f in sorted(os.listdir(root_path))
+                             if fnmatch(f, '*.xml')and fnmatch(f, 'common*')])

```

Comment:
Missing space before `and`

Reply:
There were a couple of spaces I missed. Thanks.